Formula/m/msieve.rb

```diff
@@ -0,0 +1,26 @@
+class Msieve < Formula
+  desc "C library for factoring large integers"
+  homepage "https://sourceforge.net/projects/msieve/"
+  url "https://downloads.sourceforge.net/project/msieve/msieve/Msieve%20v1.53/msieve153_src.tar.gz"
+  sha256 "c5fcbaaff266a43aa8bca55239d5b087d3e3f138d1a95d75b776c04ce4d93bb4"
+  license :public_domain
+
+  livecheck do
+    url :stable
+    regex(%r{url=.*?/Msieve%20v?(\d+(?:\.\d+)+)/}i)
+  end
+
+  depends_on "gmp"
+
+  uses_from_macos "zlib"
+
+  def install
+    inreplace "common/util.c", "#include <util.h>", "#include <util.h>\n#include <sys/time.h>"
```

Comment:
```suggestion
    ENV.append_to_cflags "-include sys/time.h"
```
This might work instead of an `inreplace`.